database/schema/01-auth.sql

```diff
@@ -1,4 +1,14 @@
 CREATE TABLE account
 (
     address TEXT NOT NULL PRIMARY KEY
+);
+
+
+CREATE TABLE vesting_account
+(
+    address             TEXT        NOT NULL REFERENCES account (address) PRIMARY KEY,
+    original_vesting    JSONB       NOT NULL DEFAULT '{}',
```

Comment:
I think it's better to use another type here. We should be able to use the `[]COINS` type instead of the `JSONB` to better support `sdk.Coins`.